heartbeat/monitors/stdfields/stdfields.go

```diff
@@ -51,10 +52,13 @@ type StdMonitorFields struct {
 		Local  *config.C `config:"local"`
 	} `config:"source"`
 	RunFrom *hbconfig.LocationWithID `config:"run_from"`
+	// Set to true by monitor.go if monitor configuration is unrunnable
+	// Maybe there's a more elegant way to handle this
+	BadConfig bool
 }
 
 func ConfigToStdMonitorFields(conf *config.C) (StdMonitorFields, error) {
-	sFields := StdMonitorFields{Enabled: true}
+	sFields := StdMonitorFields{Enabled: true, MaxAttempts: 1}
```

Comment:
I feel like we should make this configurable through env variable so we can control it on the service in a easier way than modifying the HB configs? Thoughts?

Reply:
This is a requirement for now, I'm glad to do it in a follow-up PR though! Let's see if there's a way we can do it that won't break the current Kibana UI. By requiring the config option only newer kibanas will see the new schema.